packages/react-storage/src/components/StorageManager/hooks/useStorageManager/__tests__/useStorageManager.test.ts

```diff
@@ -41,6 +41,23 @@ describe('useUploadFiles', () => {
     expect(result.current.files[1].status).toStrictEqual(FileStatus.QUEUED);
   });
 
+  it('should clear files', () => {
+    const { result } = renderHook(() => useStorageManager(defaultFiles));
+    act(() =>
+      result.current.addFiles({
+        files: [
+          new File(['test1'], 'test1.txt', { type: 'text/plain' }),
+          new File(['test2'], 'test2.txt', { type: 'text/plain' }),
+        ],
+        getFileErrorMessage: () => '',
+      })
+    );
+
+    expect(result.current.files.length).toBe(4);
```

Comment:
Does the initial value of `result.current.files.length` start at `2`?

Reply:
The `defaultFiles` contains 2 files which I passed in to the `useStorageManager` hook, plus the addition of 2 more files.